packages/gestalt/src/Text.js

```diff
@@ -106,10 +115,22 @@ export default function Text({
   underline = false,
   weight = 'normal',
 }: Props): Node {
+  const semanticColors = [
```

Comment:
suggestion

i think we could prevent mismatches if we move `semanticColors` to textTypes.js and then spread it into `allowedColors`